public/wanna-isotope-shortcode.php

```diff
@@ -24,7 +24,112 @@
  */
 class Wanna_Isotope_Shortcode {
 
-	/**
+    /**
+     * ID passed in shortcode
+     *
+     * @since 1.0.5
+     * @var int
+     */
+    private $id = 0;
+
+    /**
+     * class passed in shortcode
+     *
+     * @since 1.0.5
+     * @var string
+     */
+    private $class = '';
+
+    /**
+     * Type of post type passed in shortcode
+     *
+     * @since 1.0.5
+     * @var string
+     */
+    private $type = '';
+
+    /**
+     * count of items to be displayed passed in shortcode
+     *
+     * @since 1.0.5
+     * @var int
+     */
+    private $items = 0;
+
+    /**
+     * Order of items to be displayed passed in shortcode
+     *
+     * @since 1.0.5
+     * @var string
+     */
+    private $order = '';
+
+    /**
+     * sorting of items to be displayed passed in shortcode
+     *
+     * @since 1.0.5
+     * @var string
+     */
+    private $order_by = '';
+
+    /**
+     * category passed in shortcode
+     *
+     * @since 1.0.5
+     * @var string
+     */
+    private $tax = '';
+
+    /**
+     * Term passed in shortcode
+     *
+     * @since 1.0.5
+     * @var string
+     */
+    private $term = '';
+
+    /**
+     * Current Page number
+     *
+     * @since 1.0.5
+     * @var int
+     */
+    private $paged = 0;
+
+    /**
+     * Pagination flag
+     *
+     * @since 1.0.5
+     * @var int
+     */
+    private $pagination= '';
+
+    /**
+     * array of terms if term are passed as comma separated in shortcode
+     *
+     * @since 1.0.5
+     * @var array
+     */
+    private $terms = array();
+
+    /**
+     * array of taxonomies if tax are passed as comma separated in shortcode
+     *
+     * @since 1.0.5
+     * @var array
+     */
+    private $taxes = array();
```

Comment:
Can you avoid the taxes word? You could better use taxonomies as it's the correct term.
Taxes might be confusing. :-)